lib/mysql.js

```diff
@@ -691,10 +691,10 @@ MySQL.prototype.createTable = function (model, cb) {
  * @param {String[]} [models] A model name or an array of model names. If not present, apply to all models
  * @param {Function} [cb] The callback function
  */
-MySQL.prototype.isActual = function(cb) {
+MySQL.prototype.isActual = function(models, cb) {
   var self = this;
   var ok = false;
-  async.each(Object.keys(this._models), function(model, done) {
```

Comment:
The models arg is optional. Can you use the same logic at https://github.com/strongloop/loopback-connector-postgresql/blob/master/lib/postgresql.js#L1009-L1020?